README.md

```diff
@@ -10,19 +10,15 @@ Add the gem to your Gemfile:
 
 Run `bundle install` to install it.
 
-To add or create a user with token authentication run:
-`rails generate token_authenticate_me:install <model>`
+To install run the following:
+`rails generate token_authenticate_me:install`
 
-Replace `<model>` with the class name used for users. This will create the necessary migration files, and optionally create the model file if it does not exist.
-
-**Right now this gem only supports creating the authentication model `User`, so it is recommended to call `rails generate token_authenticate_me:install user`**
-
-Include TokenAuthenticateMe::TokenAuthentication into the application controller or any controllers that require authorization:
+Include TokenAuthenticateMe::TokenAuthenticateable into the application controller or any controllers that require authorization:
```

Comment:
I believe the correct namespace is `TokenAuthenticateMe::Concerns::Controllers::TokenAuthenticateable`